modules/nw-endpoint-route53.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-government-region.adoc
+
+[id="nw-endpoint-route53_{context}"]
+= Ingress Operator endpoint configuration for AWS Route53
+
+If you install in either Amazon Web Services (AWS) GovCloud (US) US-West or US-East region, the Ingress Operator uses `us-gov-west-1` region for Route53 and tagging API clients.
+
+The Ingress Operator uses `https://tagging.us-gov-west-1.amazonaws.com` as the tagging API endpoint if a tagging custom endpoint is configured that includes the string 'us-gov-east-1'.
+
+For more information on AWS GovCloud (US) Endpoints, see the link:https://docs.aws.amazon.com/govcloud-us/latest/UserGuide/using-govcloud-endpoints.html[AWS GovCloud (US) Service Endpoints documentation].
+
+[IMPORTANT]
+====
+Private, disconnected installs are not supported for AWS GovCloud when installing in the `us-gov-east-1` region.
+====
+
+.Example Route53 configuration
+[source,yaml]
+----
+platform:
+  aws:
+    region: us-gov-west-1
+    serviceEndpoints:
+    - name: ec2
+       url: https://ec2.us-gov-west-1.amazonaws.com
+    - name: elasticloadbalancing
+       url: https://elasticloadbalancing.us-gov-west-1.amazonaws.com
+    - name: s3
```

Comment:
the name should be `- name: route53`, others look good. Thank you.

Reply:
Thank you! I'll update that now.